MultiVersions/Fabric/main/clean-debug/lang/zh_tw.json

```
@@ -0,0 +1,33 @@
+{
+  "clean-debug.midnightconfig.title": "乾淨 F3 設定",
```

```suggestion
  "clean-debug.midnightconfig.title": "Clean F3 設定",
```
這種未新增實際遊戲內容的模組名稱，應該是不用翻